### DISCUSSION TITLE
Primevue: TieredMenu: submenu open on mouse hover and not by click

### DISCUSSION THREAD

**Discussion by MassimoBev** [primefaces/community#927] [PrimeVue]
I would like to have a props to decide if the submenus should be opened by click or mouse hover

**Comment by crackerjack-tech** [comment]
Attempting to bump this feature. Happy to fund this development work - please get in touch jack@c-j.io

**Comment by bwhitt** [comment]
Some related issue I'm running into if helps with context. If i want the main menu item to have a click action, but also have a submenu that has additional actions and options, I can't have the "click" on the main item to be "open" as well as "perform action." I would need to have "hover" be show the submenu and "click" be perform action.
